api/orders.js

```diff
@@ -155,10 +152,8 @@ function getOrders(request, response, next) {
  *  @query
  *  @param {String "true"|"false"} request.query.validated    - used to force request to wait until rippled has finished validating the submitted transaction
  *
- *  @param {Express.js Response} response
```

Comment:
Yeah, that's what I was thinking, the rest of the parameters are going to change in the next pull request.